certified-connectors/TrueDialog SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1762 @@
+      "x-ms-notification-content": {
+        "description": "Stop received",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "CallbackTimestamp": {
+              "type": "string",
+              "description": "Date and time the callback has been sent. Format: yyyy-MM-ddTHH:mm:ss",
+              "title": "Callback Timestamp"
+            },
+            "CallbackToken": {
+              "type": "string",
+              "description": "CallbackToken"
+            },
+            "CallbackType": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Always 6 for Opt-Out",
+              "title": "CallbackType",
+              "x-ms-visibility": "internal"
+            },
+            "CallbackURL": {
+              "type": "string",
+              "description": "CallbackURL",
+              "x-ms-visibility": "internal"
+            },
+            "TransactionId": {
+              "type": "string",
+              "description": "TransactionId"
+            },
+            "AccountId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "AccountId"
+            },
+            "AccountName": {
+              "type": "string",
+              "description": "The name of the account the callback is associated to",
+              "title": "Account Name"
+            },
+            "ChannelId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "ChannelId"
+            },
+            "ChannelCode": {
+              "type": "string",
+              "description": "Code of the channel the message has been sent over (the actual phone number for long codes)",
+              "title": "Channel Code"
+            },
+            "ContactId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "ContactId"
+            },
+            "ContactAccountId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "The ID of the account that owns the contact record",
+              "title": "ContactAccountId"
+            },
+            "ContactAccountName": {
+              "type": "string",
+              "description": "The name of the account that owns the contact record",
+              "title": "ContactAccountName"
+            },
+            "PhoneNumber": {
+              "type": "string",
+              "description": "The phone number of the end user that has been opted out",
+              "title": "PhoneNumber"
+            },
+            "SubscriptionId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "SubscriptionId",
+              "x-ms-visibility": "internal"
+            }
+          }
+        }
+      },
+      "put": {
+        "responses": {
+          "201": {
+            "description": "Stop callback created"
```

Comment:
@TrueDialogDynamics Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
Added schema for all the responses in [this commit](https://github.com/microsoft/PowerPlatformConnectors/pull/1337/commits/d4d6d4c7b8aa28c0eceb87f9eccd4625ec7c5d84)